admin/base/settings/defaults.py

```diff
@@ -25,18 +25,34 @@
     '.osf.io'
 ]
 
+# Email settings. Account created for testing. Password shouldn't be hardcoded
```

Comment:
@cwisecarver Ah I understand. 
Would I be able to test the password reset link from the console? Like just copy and paste it?